large-message-core/src/main/java/com/bakdata/kafka/AbstractLargeMessageConfig.java

```diff
@@ -114,6 +114,13 @@ public class AbstractLargeMessageConfig extends AbstractConfig {
                     + "recommended to enable this option.";
     public static final boolean USE_HEADERS_DEFAULT = false;
 
+    public static final String COMPRESSION_TYPE_CONFIG = PREFIX + "compression.type";
```

Comment:
please mention this in the JavaDoc of this class. You also need to update the README